near_api/account.py

```diff
@@ -25,16 +29,40 @@ def __init__(
             self,
             provider: 'near_api.providers.JsonProvider',
             signer: 'near_api.signer.Signer',
-            account_id: Optional[str] = None
+            account_id: Optional[str] = None,
+            tx_nonce_retry_number: int = 12,
```

Comment:
I copied these default number of retries from near-api-js, that's why they're so huge... (~30 retries in total with delay from 1.5s to 1min, theoretically a request can take ~3 minutes).

Reply:
With the current code request can take up to 6 minutes.  Note that JS code multiplies delay by 500 milliseconds which is not done here on line 63.